packages/sync/src/connect-webrtc.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+// @ts-ignore
+import { WebrtcProvider } from 'y-webrtc';
+
+/** @typedef {import('./types').ObjectType} ObjectType */
+/** @typedef {import('./types').ObjectID} ObjectID */
+/** @typedef {import('./types').CRDTDoc} CRDTDoc */
+
+/**
+ * Connect function to the IndexedDB persistence provider.
+ *
+ * @param {ObjectID}   objectId   The object ID.
+ * @param {ObjectType} objectType The object type.
+ * @param {CRDTDoc}    doc        The CRDT document.
+ *
+ * @return {Promise<() => void>} Promise that resolves when the connection is established.
+ */
+export function connectWebRTC( objectId, objectType, doc ) {
+	const docName = `${ objectType }-${ objectId }`;
```

Comment:
I think the document names strings should be site specific e.g.: each site has its own unique document names. It would leave the possibility open for example to use a y-websocket connector for multiple WordPress websites, or a y-webrtc signaling server for multiple sites, or even a public .org signaling server.

Reply:
Makes sense